include/common/nas_headers.h

```diff
@@ -57,6 +57,10 @@ extern "C"{
 #define NAS_DETACH_REQUEST 0x45
 #define NAS_DETACH_ACCEPT 0x46
 #define NAS_SERVICE_REQUEST 0x4D
+#define NAS_ACT_DED_BEARER_CTXT_ACPT 0xc6
+#define NAS_ACT_DED_BEARER_CTXT_RJCT 0xc7
+
+#define ESM_HEADER_LEN 9
```

Comment:
I would be interested in knowing how we arrived at this 9.

Reply:
NAS ESM Messages(Incoming) has the length of the NAS PDU Header equal to 9 Octets, distributed as below:
Security Header Type (1/2) + Protocol Discriminator (1/2) + MAC (4) + Sequence Number (1) +
EPS Bearer Identity (1/2) + Protocol Discriminator (1/2) + Procedure Transaction Identity (1) + NAS ESM Message Type (1)